src/OpenTelemetry.Api/Correlations/CorrelationContextEntry.cs

```diff
@@ -16,35 +16,35 @@
 
 using OpenTelemetry.Internal;
 
-namespace OpenTelemetry.Context
+namespace OpenTelemetry.Correlations
 {
     /// <summary>
     /// Distributed Context entry with the key, value and metadata.
     /// </summary>
-    public readonly struct DistributedContextEntry
+    public readonly struct CorrelationContextEntry
     {
         /// <summary>
-        /// Initializes a new instance of the <see cref="DistributedContextEntry"/> struct with the key and value.
+        /// Initializes a new instance of the <see cref="CorrelationContextEntry"/> struct with the key and value.
         /// </summary>
         /// <param name="key">Key name for the entry.</param>
         /// <param name="value">Value associated with the key name.</param>
-        public DistributedContextEntry(string key, string value)
-            : this(key, value, EntryMetadata.NoPropagationEntry)
+        public CorrelationContextEntry(string key, string value)
+            : this(key, value, CorrelationEntryMetadata.NoPropagationEntry)
```

Comment:
would it make sense to keep it as EntryMetadata? I guess it is possible can be used in other places outside the correlation context if we need to. what do you think?

Reply:
I left this as EmtryMetadata.